inc/xeve.h

```diff
@@ -594,6 +594,13 @@ int  XEVE_EXPORT xeve_param_default(XEVE_PARAM* param);
 int  XEVE_EXPORT xeve_param_ppt(XEVE_PARAM* param, int profile, int preset, int tune);
 int  XEVE_EXPORT xeve_param_check(const XEVE_PARAM* param);
 
+/** Set one parameter by name.
+ * @retval 0 on success
+ * @retval XEVE_ERR_INVALID_ARGUMENT
+ * @retval XEVE_ERR
+ */
```

Comment:
Currently, we're not using doxygen syntax. How about removing this comments for consistency?

Reply:
My recommendation is only for API, not internal comments.

Reply:
@kpchoi 

If everything is OK please merge the PR to the main branch.